backend/api/Services/InspectionFindingsService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Api.Database.Context;
+using Api.Database.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Api.Services
+{
+    public class IInspectionFindingService(FlotillaDbContext context)
```

Comment:
This should be just `InspectionFindingService` yeah? As the "I" part in front is to signalize this is an interface

Reply:
Yes I agree.

Reply:
Done.